### DISCUSSION TITLE
大佬能否增加V3备份导入功能

### DISCUSSION THREAD

**Discussion by zsh2708** [AlistGo/alist#1901] [General]
大佬能否增加V3备份导入功能，能V2升级最好

**Comment by 12189108** [comment]
备份是把data目录下的config.json和data.db复制出来就行，导入就是把你之前复制的再复制回原来的目录

**Reply by zsh2708** [reply]
复制进来密码会不正确，不知是否有解决方案，为何不能将备份/导入界面加到3.0呢？

**Reply by erbanku** [reply]
停止Alist，然后把/opt/alist目录整个复制下来当备份文件，以后需要恢复直接用备份目录覆盖即可。

如果想转移到新服务器，先在新服务器上通过[一键安装脚本安装Alist](https://alist.nn.ci/guide/install/script.html#:~:text=One%2Dclick%20Script-,One%2Dclick%20Script,-Noah%20Hsu)，然后用之前备份下来的Alist目录替代新服务器上的对应Alist，目录就可。

**Reply by erbanku** [reply]
这样以来你的几乎所有数据（包括密码）都可以跟之前一样了。V2升级V3没找到解决办法，我选择了重新添加存储参数。

**Comment by itsHenry35** [comment]
![image](https://user-images.githubusercontent.com/58730749/201457861-8f225269-0012-4c38-89be-bebd7824f727.png)

https://github.com/alist-org/alist-web/pull/12

**Reply by erbanku** [reply]
> ![image](https://user-images.githubusercontent.com/58730749/201457861-8f225269-0012-4c38-89be-bebd7824f727.png)
> 
> [alist-org/alist-web#12](https://github.com/alist-org/alist-web/pull/12)

我之前看到作者在一条回复中说还没开发好(正在开发），暂时找不着了。可以采用我在上面提供的方法备份、还原。
